SparkPost/2web2ui
# FE-872: Component Wrapper for Matchbox Error

### What Changed

These are the components affected.

<img width="691" alt="Screen Shot 2020-03-17 at 12 41 43 AM" src="https://user-images.githubusercontent.com/1335605/76822668-23dd4a00-67e8-11ea-8cdc-9e1c218082de.png">

### How To Test

- Pick a date range less than 7 days on any of the signals pages

<img width="1387" alt="Screen Shot 2020-03-18 at 12 33 29 AM" src="https://user-images.githubusercontent.com/1335605/76925515-1db2a080-68b0-11ea-9da8-b37cb1b72ce1.png">

- Click to browse file system, then cancel to see required error on recipient list and suppression list.

<img width="1390" alt="Screen Shot 2020-03-18 at 12 35 39 AM" src="https://user-images.githubusercontent.com/1335605/76925629-6ff3c180-68b0-11ea-8980-98fc1f2f9379.png">

- Type some alert name on create alert page then click submit button
<img width="1389" alt="Screen Shot 2020-03-18 at 12 41 16 AM" src="https://user-images.githubusercontent.com/1335605/76925849-32dbff00-68b1-11ea-894e-eb13ce0b858f.png">
